peerconnection.go

```diff
@@ -1950,6 +1950,34 @@ func (pc *PeerConnection) GetStats() StatsReport {
 	pc.mu.Unlock()
 
 	statsCollector.Collect(stats.ID, stats)
+
+	if pc.api.mediaEngine != nil {
+		for _, sender := range pc.GetSenders() {
```

Comment:
You can use `GetTransceivers` to get both senders and receivers

Reply:
Actually, I dont think this should be derived from the transceivers. It should just be generated based on the current `MediaEngine`. You can define a method `MediaEngine.collectStats` that aggregates codec stats then is called here with `pc.api.mediaEngine.collectStats(...)`